package.json

```
@@ -2,24 +2,36 @@
   "name": "network-checker",
   "version": "0.1.1",
   "description": "A module for checking/monitoring a URL",
-  "main": "index.js",
-  "typings": "index.d.ts",
+  "main": "dist/index.js",
   "scripts": {
-    "prepublish": "coffee -c index.coffee",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "prettify": "balena-lint --fix lib/ test/",
```

Could you rename this to `lint-fix` as that's what we've started to standardize on:
```suggestion
    "lint-fix": "balena-lint --fix lib/ test/",
```

---

Done!